astropy/constants/codata2014.py

```diff
@@ -90,7 +90,7 @@ class EMCODATA2014(CODATA2014, EMConstant):
                      system='si')
 
 b_wien = CODATA2014('b_wien', 'Wien wavelength displacement law constant',
-                    2.8977729e-3, 'm K', 00.0000017e-3, system='si')
+                    2.8977729e-3, 'm K', 0.000000017e-3, system='si')
```

Comment:
In the CODATA references this is written as `2.897 7729(17)` (times 10**-3) which is five zeroes after the decimal point, if I am reading this correctly. 0.0000017e-3.

Reply:
Yes, based on the notation at https://physics.nist.gov/cgi-bin/cuu/Value?mp|search_for=proton+mass , you are right! Sorry, I'll remove the change log then and clear things up.